open-api/rest-catalog-open-api.yaml

```diff
@@ -2033,6 +2537,68 @@ components:
         metadata-location:
           type: string
 
+    CreateViewRequest:
+      type: object
+      required:
+        - name
+        - schema
+        - view-version
+        - properties
+      properties:
+        name:
+          type: string
+        location:
+          type: string
+        schema:
+          $ref: '#/components/schemas/Schema'
+        view-version:
+          $ref: '#/components/schemas/ViewVersion'
```

Comment:
This will need to replace the `schema-id` sent with the `ViewVersion` with whatever was assigned to the schema above.

Reply:
I've added a description to make this clearer